### PR TITLE
Update to migrations

### LINKED ISSUE #1101: Make `linked_civis` and `response_civis` fields optional on Civi model
The `Civi` model contains two required fields that are causing difficulty when starting a fresh project:

- `linked_civis` https://github.com/CiviWiki/OpenCiviWiki/blob/85cd5bf867be4e1987cff3f083fc456904287f9e/project/threads/models.py#L214
- `response_civis` https://github.com/CiviWiki/OpenCiviWiki/blob/85cd5bf867be4e1987cff3f083fc456904287f9e/project/threads/models.py#L215-L221

Since those fields are required, it is not possible to create the first Civi, due to the "chicken and egg" situation where no Civis exist.

## Task

- [x] remove the `response_civis` field
- [x] make the `linked_civis` field optional
- [x] create database migrations with `python manage.py makemigrations`

## Original report
@AbhijithGanesh opened this issue originally with the following information.

### Description

Without fixtures in Civi, it is not possible to create Civis and it is requires a civi to be referenced when created, which makes it impossible to create.

### PR DESCRIPTION
**Pull request by AbhijithGanesh**
Closes #1101 
  
## Description
Resolves the necessary changes to models and creates migrations

### CODE REVIEW

**Review by gorkemarslan — APPROVED** [review]

**Comment by AbhijithGanesh** [comment]
@brylie can this be merged?

**Comment by gorkemarslan** [comment]
@AbhijithGanesh, The changes are fine to me. I am waiting for @brylie's response. They asked a question in issue #1101 and I don't know whether they are convinced or have anything else to add. This is the reason I have not merged this PR yet. 

I try not to be impetuous. I'm planning to merge only the obvious bug fixes and the PRs that @brylie approves. @brylie is a vision holder on this project. I just try to learn their project management skills. 👍

File: project/threads/migrations/0002_auto_20211012_2014.py [hunk 1]
```diff
@@ -1,49 +1,58 @@
...
+        migrations.AlterField(
+            model_name='response',
+            name='author',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='thread',
+            name='author',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL),
+        ),
```
Comment (brylie) [inline project/threads/migrations/0002_auto_20211012_2014.py:57]: @gorkemarslan, are these migrations reverting changes you made recently? Specifically, regarding the User/author model relationships?

Comment (gorkemarslan) [reply project/threads/migrations/0002_auto_20211012_2014.py:57]: @brylie, no, they all are the same as [my changes in migrations](https://github.com/CiviWiki/OpenCiviWiki/pull/1110/commits/f8430d621ad1cda8cf4c4f87fc4afc161cda1668). I guess AbhijithGanesh  deleted migration file and made migrations then. As a result of deletion and creation of a file, Git perceived these lines as new changes.

**Pull request merged**